services/ui-src/src/utils/fileTypes.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+
+interface FileTypeInfo {
+  extension: string;
+  description: string;
+}
+/** The source of truth for filetypes accepted as attachments in forms. */
+export const FILE_TYPES: FileTypeInfo[] = [
```

Comment:
This list does not match the allow list from serverless.yml in uploads directory.  I see this is what the Figma has, but there is absolutely NO WAY we are allowing .dmg files to be uploaded without explicit instructions to do so.

Reply:
HA! Yes, `.dmg` had me _so curious_! I was like "idk, maybe it's a gov sec thing" 😂 I'll reference the `serverless.yml` -- we have a few of those right; which directory has that specific one?